projects/plugins/crm/includes/ZeroBSCRM.DAL3.Obj.Segments.php

```diff
@@ -1435,7 +1435,24 @@ public function segmentConditionArgs($condition=array(),$conditionKeySuffix=''){
 
                     global $zbs,$wpdb,$ZBSCRM_t;
 
-                    switch ( $condition['type'] ){
+					$advanced_segments_active = defined( 'JPCRM_ADVANCED_SEGMENTS_ROOTFILE' );
```

Comment:
Just dropping my two cents here: personally I dislike the idea of the core plugin referencing extensions. I think that the core plugin should not be aware of the existence of specific extensions, and should be prepared to receive input from any code that calls a filter/action/hook.

Not a blocker though.

Reply:
I don't think we need check at all; we're just changing the order of precedence (non-core, then core).

Reply:
In fact, adding the check actually breaks conditions added by the core Woo module.

Reply:
Ah interesting, thanks for checking this.

Reply:
Fixed in 07a1fd2.